File: Frontend/HttpClient/src/app/models/Reporting/transport-operation-block.ts
export class TransportOperationBlock {
    public id: number;

    public firstWorkshopId: number;

    public firstWorkshopName: string;

    public secondWorkshopId: number;

    public secondWorkshopName: string;

    public distance: number;

    public duration: number;

    static fromJSON(obj: any) {
        if (!obj) {
            return null;
        }

        return Object.assign(
            new TransportOperationBlock(),
            obj
        );
    }
}

File: Backend/Scheduler/Model/OrderReport/TransportOperationBlock.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Scheduler.Model.OrderReport
{
    public class TransportOperationBlock
    {
        public int Id { get; set; }

        public int FirstWorkshopId { get; set; }
        public string FirstWorkshopName { get; set; }

        public int? SecondWorkshopId { get; set; }
        public string SecondWorkshopName { get; set; }

        public float Distance { get; set; }

        public long Duration { get; set; }

        public int GroupBlockId { get; set; }
        public virtual GroupBlock GroupBlock { get; set; }
    }
}
